README.md

```diff
@@ -28,8 +28,8 @@ This plugin can be installed from the [Update Center][3] (found at `Manage Jenki
 
 There are two ways to configure your plugin to submit data to Datadog:
 
-* **RECOMMENDED**: Using a DogStatsD server / Datadog Agent that acts as a forwarder between Jenkins and Datadog.
-  - Build Logs collection only works with a full Datadog Agent installed.
+* **RECOMMENDED**: Using a Datadog Agent that acts as a forwarder between Jenkins and Datadog.
+  - Build Logs collection and CI Visibility only work with a full Datadog Agent installed.
```

Comment:
If you are asking for a full Datadog Agent, then this line doesn't apply. If anything I would suggest:

```suggestion
  - When using a DogStatsD server instead of a full Datadog Agent, only metrics and events are supported.
```